drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -247,9 +247,6 @@ void RasterizerGLES3::set_current_render_target(RID p_render_target) {
 
 	if (p_render_target.is_valid()) {
 		RasterizerStorageGLES3::RenderTarget *rt = storage->render_target_owner.getornull(p_render_target);
-		if (!rt) {
-			storage->frame.current_rt = NULL;
-		}
 		ERR_FAIL_COND(!rt);
 		storage->frame.current_rt = rt;
```

Comment:
I'm not sure if the new behaviour would be correct, but to get the old behaviour, you should move this line above ERR_FAIL_COND.
```cpp
storage->frame.current_rt = rt;
ERR_FAIL_COND(!rt);
```